docs/features/pouch_with_lxcfs.md

```diff
@@ -6,6 +6,29 @@ Container solutions provide a general way for users to limit resource usage of r
 
 For example, if we create a general container with limiting 200 MB memory on host of 2 GB memory in total, we can find that resource upper limit is incorrect via command `free`, and it shows all memory size of the host:
 
+Make sure your lxcfs service is running:
+
+```
+$ systemctl start lxcfs
```

Comment:
Maybe we should add some restriction for this part code shell command, since I think `systemctl` can be used on some constraint os distributions, not all. 

Otherwise, doc readers may feel that for all the os distributions, they could use these script command. WDYT? @fanux